README.md

```diff
@@ -398,56 +398,17 @@ Browse the files in the [/example](https://github.com/zoontek/react-native-local
 
 ## How to test your code
 
-Because it's a native module, you might need to mock this package to run your tests flawlessly.<br />
-Here is an example for Jest, adapt it to your needs :
-
+Because it's a native module, you need to mock this package in jest to run your tests.<br />
+The package provides a default mock you may reference in your jest `setupFiles` config as:
+
+````json
+  "setupFiles": [
+    // ...other setup files here
+    "./node_modules/react-native-localize/jestSetup.js"
```

Comment:
LGTM, but could you rename the file `mock.js`? For consistency with rn-permissions :)

Reply:
💥  named to mock.js, updated package.json::files section and readme ref, re-pushed as single commit